certification/internal/bundle/bundle.go

```diff
@@ -198,16 +198,66 @@ func GetSupportedInstallModes(ctx context.Context, csvReader io.Reader) (map[str
 	return installedModes, nil
 }
 
+func ExtractImagesFromBundle(ctx context.Context, csvReader io.Reader) ([]string, error) {
+	csvBytes, err := io.ReadAll(csvReader)
+	if err != nil {
+		return nil, fmt.Errorf("could not read CSV from path: %v", err)
+	}
+
+	var csv ClusterServiceVersion
+	if err := yaml.Unmarshal(csvBytes, &csv); err != nil {
+		return nil, fmt.Errorf("could not unmarshal CSV: %v", err)
+	}
+
+	imageMap := make(map[string]struct{}, len(csv.Spec.RelatedImages))
```

Comment:
Do we know if the pipeline updates the `relatedImages` section in the bundle that we get? Or do they just update it for their own use, and we get the raw csv that the partner put in their PR that might not have `relatedImages`?

Reply:
Yeah I have no idea either, I was just wondering, since this could be the length of RelatedImages could be zero. Not really a blocker for this PR or a code change, but we should probably know that when we go to add this check in and fail certification.